File: 10(Summation of primes)_D5.py
import math
numInput = 2000000
cycle = 1
start = 1
number = 0
while cycle < numInput:
    start +=1
    sqrt_start = math.sqrt(start)
    sqrt_start = int(sqrt_start)
    for i in range(2, sqrt_start):          
        if start != i: 
            if (start % i) == 0:
                break
    else:
        number += start
    cycle+=1
print('big nibba' + str(number))

File: Javascript/10(Summation of primes)_D5.js
var number = 2;
var sum = 0;
for(number = 2; number<2000000;){
    var postMath = Math.floor(Math.sqrt(number));
    if(postMath != 1){
        if(Math.sqrt(number) % 1 != 0){
            var broke = false;
            for(cycle = 2; cycle <= postMath;){
                if(number % cycle == 0){
                    broke = true;
                    break;
                }else{
                    cycle++;
                }
            }
            if(broke == false){
                sum+= number;
            }
        }
    }else{
        sum+= number;
    }
    number++;
}
console.log(sum)